doc/changes/changes_0.5.0.md

```diff
@@ -0,0 +1,22 @@
+# 0.5.0 - 2024-10-24
+
+## Fixes
+
+* [#27](https://github.com/exasol/error-reporting-python/issues/27) Make sure creating an error does not raise an exception.
```

Comment:
we usually don't add links

Reply:
Why not having the link makes it more valuable imho -> helps quickly checking it out